app/src/main/java/fr/neamar/kiss/result/SearchResult.java

```diff
@@ -48,6 +48,7 @@ public View display(Context context, int position, View v) {
     public void doLaunch(Context context, View v) {
         boolean exceptionThrown = false;
         Intent search = new Intent(Intent.ACTION_WEB_SEARCH);
+		search.setSourceBounds(v.getClipBounds());
```

Comment:
I have a crash here when running your version and searching for a string + pressing "enter" on the keyboard (not clicking on the result):

```
                          Process: fr.neamar.kiss, PID: 29611
                                                                java.lang.NullPointerException: Attempt to invoke virtual method 'android.graphics.Rect android.view.View.getClipBounds()' on a null object reference
                                                                    at fr.neamar.kiss.result.SearchResult.doLaunch(SearchResult.java:51)
                                                                    at fr.neamar.kiss.result.Result.launch(Result.java:165)
                                                                    at fr.neamar.kiss.adapter.RecordAdapter.onClick(RecordAdapter.java:81)
                                                                    at fr.neamar.kiss.MainActivity$6.onEditorAction(MainActivity.java:284)
                                                                    at android.widget.TextView.onEditorAction(TextView.java:4903)
                                                                    at com.android.internal.widget.EditableInputConnection.performEditorAction(EditableInputConnection.java:138)
                                                                    at com.android.internal.view.IInputConnectionWrapper.executeMessage(IInputConnectionWrapper.java:372)
                                                                    at com.android.internal.view.IInputConnectionWrapper$MyHandler.handleMessage(IInputConnectionWrapper.java:93)
                                                                    at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                    at android.os.Looper.loop(Looper.java:154)
                                                                    at android.app.ActivityThread.main(ActivityThread.java:6119)
                                                                    at java.lang.reflect.Method.invoke(Native Method)
                                                                    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
                                                                    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```